var/spack/repos/builtin/packages/py-metomi-rose/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMetomiRose(PythonPackage):
+    """Rose, a framework for meteorological suites."""
+
+    homepage = "https://metomi.github.io/rose/doc/html/index.html"
+    pypi = "metomi-rose/metomi-rose-2.1.0.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("2.1.0", sha256="1b60135a434fe4325d364a57e8f5e81e90f39b373b9d68733458c1adc2513c05")
+
+    depends_on("python@3.7:", type=("build", "run"))
```

Comment:
Optional: spack only supports python@3.7: anyway, so this dependency is not needed.